### DISCUSSION TITLE
Is there any plan for this Naive UI to be made for React.js as well?

### DISCUSSION THREAD

**Discussion by Deri-Kurniawan** [tusen-ai/naive-ui#6081] [General]
I see that Naive UI provides many components and is very interesting. Are there any plans to make Naive UI available for the React.js library, similar to how ShadcnUI is available for React.js and Vue.js?

Thanks!

**Comment by jahnli** [comment]
很遗憾目前没有，https://github.com/tusen-ai/naive-ui/issues/2223
